lib/src/appcast.dart

```diff
@@ -36,6 +36,34 @@ class Appcast {
   late IosDeviceInfo _iosInfo;
   String? osVersionString;
 
+  /// Returns the latest item in the Appcast, including only critical updates.
+  AppcastItem? criticalUpdateItem() {
+    if (items == null) {
+      return null;
+    }
+
+    AppcastItem? bestItem;
+    items!.forEach((AppcastItem item) {
+      if (item.hostSupportsItem(osVersion: osVersionString) &&
+          item.isCriticalUpdate) {
+        if (bestItem == null) {
+          bestItem = item;
+        } else {
+          try {
+            final itemVersion = Version.parse(item.versionString!);
+            final bestItemVersion = Version.parse(bestItem!.versionString!);
+            if (itemVersion > bestItemVersion) {
+              bestItem = item;
+            }
+          } on Exception catch (e) {
+            print('appcast.bestItem: invalid version: $e');
```

Comment:
All upgrader print statements need to start with `upgrader: `

Reply:
Since this try/catch wraps two `Version.parse` statements which could throw, the print statement should not say `bestItem` when it could be `item`.

Reply:
Updated in https://github.com/larryaasen/upgrader/pull/275/commits/2783d3f700d75dfcda0f3e329a44a7495e0f2741